elk/utils_evaluation/parser.py

```diff
@@ -44,14 +44,25 @@ def get_args(default_config_path=Path(__file__).parent / "default_config.json"):
     )
     parser.add_argument("--language_model_type", type=str, default="encoder")
     parser.add_argument("--layer", type=int, default=-1)
-    parser.add_argument("--zero", type=str, default="results")
+    parser.add_argument(
+        "--zero", type=str, default="results"
+    )  # TODO: explain this. Never used.
     parser.add_argument("--seed", type=int, default=0)
     parser.add_argument(
         "--model_device",
         type=str,
         default="cuda",
         help="What device to load the model onto: CPU or GPU or MPS.",
     )
+    parser.add_argument(
+        "--use_lbfgs", action="store_true", help="Use lbfgs optimizer to train CCS."
```

Comment:
I used an enum for type safety. Literal["adam", "lbfgs"] can't be used in argparse.
But maybe you think it's better to use str and then assert that the optimizer is correct?

Reply:
You can use `choices=("adam", "lbfgs")` in `argparse`. I tend not to use enums because they're verbose— you have to type `Optimizer.adam` instead of just `"adam"`. For now I'd recommend just using strings + an `assert` statement. But maybe @lauritowal disagrees.

Reply:
Done. Also, the assert is done by argparse in the parser, and I had already an assert in the training code for ccs, so I guess no additional assert is needed.